release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1814,9 +1814,42 @@ For more information, see xref:../networking/metallb/metallb-upgrading-operator.
 
 * The OpenShift CLI (`oc`) for {product-title} 4.11 does not work properly on macOS due to a change in error handling of untrusted certificates in Go 1.18 libraries. Due to this change, `oc login` and other `oc` commands can fail with a `certificate is not trusted` error without proceeding further when running on macOS. Until the error handling is properly fixed in Go 1.18 (tracked by link:https://github.com/golang/go/issues/52010[Go issue #52010]), the workaround is to use the {product-title} 4.10 `oc` CLI instead. Note that when using the {product-title} 4.10 `oc` CLI with an {product-title} 4.11 cluster, it is no longer possible to get a token by using the `oc serviceaccounts get-token <service_account>` command. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097830[*BZ#2097830*]) (link:https://bugzilla.redhat.com/show_bug.cgi?id=2109799[*BZ#2109799*])
 
+
 * There is currently a known issue in the *Add Helm Chart Repositories* form to extend the Developer Catalog of a project. The *Quick Start* guides shows that you can add the `ProjectHelmChartRepository` CR in the desired namespace whereas it does not mention that to perform this you need permission from the kubeadmin. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054197[BZ#2054197])
 * There is currently a known issue. If you create an instance of `ProjectHelmChartRepository` custom resource (CR) that uses TLS verification, you cannot list the repositories and perform Helm-related operations. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])
 
+* In {product-title} {product-version}, the `restricted` pod security admission profile is enabled for all namespaces by default. As a result, an error dispalys if you install a catalog source that was built with an `opm` version earlier than 1.23.0.
++
+.Example error
+[source,terminal]
+----
+Error: open ./db-xxxx: permission denied
+----
++
+Workaround: Cluster administrators can allow the creation of catalog sources built with earlier `opm` versions by setting the pod security admission labels in the namespace metadata files. Apply the `baseline` label to the `warn`, `audit`, and `enforce` modes, as shown in the following example:
++
+.Example `manifests/<target_namespace>.yaml` file
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+...
+  labels:
```

Comment:
We should also add this label `security.openshift.io/scc.podSecurityLabelSync: "false"`, otherwise, they will be reset to `restricted`. Like below:
```yaml
apiVersion: v1
kind: Namespace
metadata:
...
  labels:
    security.openshift.io/scc.podSecurityLabelSync: "false"
    pod-security.kubernetes.io/enforce: baseline
    pod-security.kubernetes.io/audit: baseline
    pod-security.kubernetes.io/warn: baseline
  name: test
```

Reply:
@jianzhangbjz Addressed. PTAL!